skl2onnx/operator_converters/support_vector_machines.py

```diff
@@ -244,10 +244,10 @@ def convert_sklearn_svm_classifier(
         #     transformed_confidences = (
         #         sum_of_confidences / (3 * (np.abs(sum_of_confidences) + 1)))
         #     return votes + transformed_confidences
-        if list(op.classes_) != list(range(len(op.classes_))):
-            raise RuntimeError(
-                "Classes different from first n integers are not supported "
-                "in SVC converter.")
+        # if list(op.classes_) != list(range(len(op.classes_))):
```

Comment:
I think it is better to remove the test rather than leaving it but commented out.  It could be great if you could add a unit test about that specific case.

Reply:
I remove `if test` and change the title. Unit test haven't done yet